src/pages/SearchResultPage.js

```diff
@@ -0,0 +1,50 @@
+import PageTitle from 'components/Page Title/PageTitle';
+import SearchBar from 'components/SearchBar/SearchBar';
+import ShopListContent from 'components/ShopListContent/ShopListContent';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { useSearchParams } from 'react-router-dom';
+import {
+  searchResultProducts,
+  shoplistSortProduct,
+} from 'store/actions/common';
+import { useGetPopularProduct } from 'store/selectors/common';
+
+function SearchResult() {
+  const [searchParams] = useSearchParams();
+  const keyword = searchParams.get('query');
+  const defaultProducts = useGetPopularProduct();
+  const [responsedProducts, setResponsedProducts] = useState();
+  const dispatch = useDispatch();
+
+  const productResults = !responsedProducts
+    ? defaultProducts
+    : responsedProducts;
+
+  React.useEffect(() => {
+    fetch(
+      `https://vnguyen.xyz/huy/day17/apis/index.php?type=search&query=${keyword}`
+    )
+      .then((res) => res.json())
+      .then((data) => setResponsedProducts(data.data));
+  }, [keyword]);
+  React.useEffect(() => {
```

Comment:
1. Just a minor issue but we shouldn't usually use `React.useEffect`, just import it like normal and use like `useEffect`
2. In this part of the code, we use two `useEffect` which is unnecessary in this case, let's refactor this. The goal is to use only 1 `useEffect` but still do 2 jobs.